src/Playwright/Playwright.cs

```diff
@@ -22,26 +22,68 @@
  * SOFTWARE.
  */
 
+using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
 using Microsoft.Playwright.Core;
+using Microsoft.Playwright.Helpers;
 using Microsoft.Playwright.Transport;
 
 namespace Microsoft.Playwright
 {
     [SuppressMessage("Microsoft.Design", "CA1724", Justification = "Playwright is the entrypoint for all languages.")]
     public static class Playwright
     {
+        private static Process _playwrightServerProcess;
+
         /// <summary>
         /// Launches Playwright.
         /// </summary>
         /// <returns>A <see cref="Task"/> that completes when the playwright driver is ready to be used.</returns>
         public static async Task<IPlaywright> CreateAsync()
         {
-            var connection = new Connection();
+            var loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.SetMinimumLevel(LogLevel.Debug);
+                builder.AddFilter((f, _) => f == "PlaywrightSharp.Playwright");
+            });
+            _playwrightServerProcess = GetProcess();
+            _playwrightServerProcess.StartInfo.Arguments = "run-driver";
+            _playwrightServerProcess.Start();
+            _playwrightServerProcess.Exited += (_, _) => Close();
+
+            var transport = new StdIOTransport(_playwrightServerProcess, loggerFactory);
+            var connection = new Connection(transport, loggerFactory);
             var playwright = await connection.WaitForObjectWithKnownNameAsync<PlaywrightImpl>("Playwright").ConfigureAwait(false);
             playwright.Connection = connection;
             return playwright;
         }
+
+        private static Process GetProcess()
```

Comment:
This isn't the right place for this. The responsibility of the `Playwright` class should be incredibly simple. My first thought is to move the creation (and ownership) of the process into the `transport` class - because they're the only ones actually caring about it. So, basically, we'd end up with

`new Connection(new StdIOTransport())` 

or

`new Connection(new WebSocketTransport())`

and then the rest happens automagically.